core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -1886,6 +1886,30 @@ public String toString() {
           .setDescription("The number of workers to run on an Alluxio host for YARN framework.")
           .build();
 
+  //
+  // JVM related properties
+  //
+  public static final PropertyKey JVM_MONITOR_WARN_THRESHOLD_MS =
```

Comment:
Could you change these properties to be unit agnostic?

Reply:
Could you make an simple explanation on `unit agnostic` please? Thanks!

Reply:
For example the string value of `SECURITY_AUTHENTICATION_SOCKET_TIMEOUT_MS` is `alluxio.security.authentication.socket.timeout` (note there is no `.ms` at the end) and the default value is `10min` instead of a raw number.